modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -4070,6 +4070,18 @@ AUI.add(
 					var fields = parentField.get('fields');
 
 					fields.splice(newIndex, 0, fields.splice(oldIndex, 1)[0]);
+
+					if(CKEDITOR) {
```

Comment:
Hi @orsolyalukacs,

Can you please add a blank space after the `if`?

Additionally, would be interesting you do this check based on the field type instead of check if the `CKEDITOR` variable is defined, otherwise the code inside this statement will be executed every time some field is dragged and dropped even if this field doesn't use the CKEditor. By the way, check the existence of the variable `CKEDITOR` is the best way to check the editor's type? I looked for something similar in the portal code and I couldn't find it. 😞 

Lastly, can you please add all this logic into a separated method, like `recreateEditor`, for instance? It's important to make each method have its own responsibility. 🙂